src/queues/initialize.ts

```diff
@@ -71,8 +81,11 @@ const createQueue = async <N extends JobName>(
     },
   );
 
-  cleanup.push(() => queue.close());
-  cleanup.push(() => worker.close());
+  cleanup.push(async () => {
```

Comment:
could we call `handleExit` with this directly now instead of using the `cleanup` array?

Reply:
unfortunately, we still need the flow producer and redis connection to be shut down last